frontend/src/utilities/valueUnits.ts

```diff
@@ -155,3 +155,17 @@ export const isMemoryLimitLarger = (
 
   return isLarger(limitMemory, requestMemory, MEMORY_UNITS_FOR_PARSING, isEqualOkay);
 };
+
+export const formatMemory = (value?: string | ValueUnitString): ValueUnitString => {
+  if (!value) {
+    return 'Unknown';
+  }
```

Comment:
As a utility function, I prefer to not impose an opinionated return value. The consumer can choose to render an alternative string in place of empty or undefined.
```suggestion
export const formatMemory = (value?: ValueUnitString): ValueUnitString => {
  if (!value) {
    return value;
  }
```